### ISSUE TITLE
New domain for t411, now t411.li

### ISSUE THREAD

**Issue by bilyboy785** [SickRage/SickRage#2640]
T411 change his domain and it's now t411.li. Daily search can't find anything due probably to this change

Branch/Commit: Master/Origin
OS: Debian 8
What you did:
What happened:
What you expected:
Logs:
```
2016-11-22 20:53:42 SEARCHQUEUE-BACKLOG-121361 :: [T411] :: Token not found in authentication response
2016-11-22 08:23:11 SEARCHQUEUE-DAILY-SEARCH :: [T411] :: Unable to connect to provider
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by msoulard** [comment]
Fixed by commit a1bf3b30a9a0d6319c09fb0cf48d2d468dbf4e61, you need to update.
